docs/outlook/mobile-log-appointments.md

```diff
@@ -0,0 +1,553 @@
+                  <bt:Image size="25" scale="3" resid="icon-16"/>
+
+                  <bt:Image size="32" scale="1" resid="icon-32"/>
+                  <bt:Image size="32" scale="2" resid="icon-32"/>
+                  <bt:Image size="32" scale="3" resid="icon-32"/>
+
+                  <bt:Image size="48" scale="1" resid="icon-48"/>
+                  <bt:Image size="48" scale="2" resid="icon-48"/>
+                  <bt:Image size="48" scale="3" resid="icon-48"/>
+                </Icon>
+                <Action xsi:type="ShowTaskpane">
+                  <SourceLocation resid="Taskpane.Url"/>
+                </Action> 
+              </Control>
+            </ExtensionPoint>
+          </MobileFormFactor>
+        </Host>
+      </Hosts>
+      <Resources>
+        <bt:Images>
+          <bt:Image id="icon-16" DefaultValue="https://contoso.com/assets/icon-16.png"/>
+          <bt:Image id="icon-32" DefaultValue="https://contoso.com/assets/icon-32.png"/>
+          <bt:Image id="icon-48" DefaultValue="https://contoso.com/assets/icon-48.png"/>
+          <bt:Image id="icon-80" DefaultValue="https://contoso.com/assets/icon-80.png"/>
+        </bt:Images>
+        <bt:Urls>
+          <bt:Url id="residFunctionFile" DefaultValue="https://contoso.com/commands.html"/>
+          <bt:Url id="Taskpane.Url" DefaultValue="https://contoso.com/taskpane.html"/>
+        </bt:Urls>
+        <bt:ShortStrings>
+          <bt:String id="residDescription" DefaultValue="Log appointment notes and other details to Contoso CRM."/>
+          <bt:String id="residLabel" DefaultValue="Log to Contoso CRM"/>
+        </bt:ShortStrings>
+        <bt:LongStrings>
+          <bt:String id="residTooltip" DefaultValue="Log notes to Contoso CRM for this appointment."/>
+        </bt:LongStrings>
+      </Resources>
+    </VersionOverrides>
+</VersionOverrides>
+```
+
+> [!TIP]
+> To learn more about manifests for Outlook add-ins, see [Outlook add-in manifests](manifests.md) and [Add support for add-in commands for Outlook Mobile](add-mobile-support.md).
+
+### Implement capturing appointment notes
+
+In this section, learn how to display the logged appointment notes and other details in a task pane when the user selects the **Log** button.
+
+1. From the same quick start project, open the file **./src/taskpane/taskpane.js** in your code editor.
+
+1. Replace the entire content of the **taskpane.js** file with the following JavaScript.
+
+    ```js
+    // Office is ready.
+    Office.onReady(function () {
+        getEventData();
+      }
+    );
+
+    function getEventData() {
+      console.log(`Subject: ${Office.context.mailbox.item.subject}`);
+      Office.context.mailbox.item.body.getAsync(
+        "html",
+        function callback(result) {
+          if (result.status === Office.AsyncResultStatus.Succeeded) {
+            console.log("event logged successfully");
+            event.completed({ allowEvent: true });
```

Comment:
I think you will be changing the code snippets for taskpane addin as per the doc as we don't have event object in taskpane addin.